examples/lists_example.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Test for list structures in PyLaTeX.
+# More info @ http://en.wikibooks.org/wiki/LaTeX/List_Structures
+
+from pylatex import Document, Section, Itemize, Enumerate, Description
+
+
+doc = Document()
+
+# create a bulleted "itemize" list like the below:
+# \begin{itemize}
+#   \item The first item
+#   \item The second item
+#   \item The third etc \ldots
+# \end{itemize}
+
+with doc.create(Section('"Itemize" list')):
+    with doc.create(Itemize()) as enum:
+        enum.add_item("the first item")
+        enum.add_item("the second item")
```

Comment:
Could you maybe add an append as well? To show that it is also possible to add stuff without creating a new item?